ci/deps/actions-310.yaml

```diff
@@ -22,36 +22,36 @@ dependencies:
   - pytz
 
   # optional dependencies
-  - beautifulsoup4
+  - beautifulsoup4>=4.9.3
   - blosc
-  - bottleneck
-  - brotlipy
-  - fastparquet
-  - fsspec
-  - html5lib
-  - hypothesis
-  - gcsfs
-  - jinja2
-  - lxml
+  - bottleneck>=1.3.2
+  - brotlipy>=0.7.0
+  - fastparquet>=0.6.3
+  - fsspec>=2021.07.0
+  - html5lib>=1.1
+  - hypothesis>=6.34.2
+  - gcsfs>=2021.07.0
+  - jinja2>=3.0.0
+  - lxml>=4.6.3
   - matplotlib>=3.6.1, <3.7.0
-  - numba
-  - numexpr
-  - openpyxl<3.1.1
-  - odfpy
-  - pandas-gbq
-  - psycopg2
-  - pymysql
-  - pytables
-  - pyarrow
-  - pyreadstat
-  - python-snappy
-  - pyxlsb
+  - numba>=0.53.1
+  - numexpr>=2.7.3
+  - openpyxl>=3.0.7, <3.1.1
+  - odfpy>=1.4.1
+  - pandas-gbq>=0.15.0
+  - psycopg2>=2.8.6
+  - pymysql>=1.0.2
+  - pytables>=3.6.1
+  - pyarrow>=7.0.0
+  - pyreadstat>=1.1.2
+  - python-snappy>=0.6.0
+  - pyxlsb>=1.0.8
   - s3fs>=2021.08.0
-  - scipy
-  - sqlalchemy
-  - tabulate
-  - tzdata>=2022a
-  - xarray
-  - xlrd
-  - xlsxwriter
-  - zstandard
+  - scipy>=1.7.1
+  - sqlalchemy<1.4.46, >=1.4.16
+  - tabulate>=0.8.9
+  - tzdata>=2022.1
```

Comment:
now that you've excluded tzdata, you can change it back to `2022a` (here and in the other yaml files, as long as it's not in a `pip` section)

Reply:
Okay, done. I see some failing pyarrow on the CI. 

Should I add back `pyarrow>=7.0.0` to fix this? 
```
  error    libmamba Could not solve for environment specs
      Encountered problems while solving:
        - nothing provides requested pyarrow =9>=7.0.0
```